fdbserver/fdbserver.actor.cpp

```diff
@@ -1842,6 +1842,21 @@ int main(int argc, char* argv[]) {
 		// Reinitialize knobs in order to update knobs that are dependent on explicitly set knobs
 		g_knobs.initialize(Randomize::True, role == ServerRole::Simulation ? IsSimulated::True : IsSimulated::False);
 
+		if (!SERVER_KNOBS->ALLOW_DANGEROUS_KNOBS) {
+			if (SERVER_KNOBS->FETCH_USING_STREAMING) {
+				fprintf(stderr,
+				        "ERROR : explicitly setting FETCH_USING_STREAMING is dangerous! set ALLOW_DANGEROUS_KNOBS to "
```

Comment:
@sfc-gh-etschannen Isn't it very specific to disk types with IOPS limitation (like EBS). For disks like NVMeRAM also do we consider this dangerous? May be worth adding a comment why we think this is dangerous.

Reply:
because we do not current what caused the memory corruption, it is very possible the problem is related to ReplyPromiseStream, which is used by that feature. I am not disabling it because of EBS